app/frontend/src/features/BugReport.test.tsx

```diff
@@ -151,15 +143,14 @@ describe("BugReport", () => {
 
       expect(await screen.findByRole("progressbar")).toBeVisible();
       const successMessage =
-        "Thank you for reporting that bug and making Couchers better, a report was sent to the devs! The bug ID is 1";
+        "Thank you for reporting that bug and making Couchers better, a report was sent to the developers! The bug ID is 1";
```

Comment:
Use the `getBugReportSuccessMessage` you made in the constants file to simplify this

Reply:
OK